facebook/delphiFacebook/R/binary.R

```diff
@@ -105,6 +105,7 @@ compute_binary_response <- function(response, weight, sample_size)
 #' @param df Data frame
 #' @return Updated data frame.
 #' @importFrom dplyr mutate
+#' @export
```

Comment:
if we move table configuration into the library, we won't need to export these anymore